proto/axelar/evm/v1beta1/events.proto

```diff
@@ -115,3 +115,19 @@ message ChainAdded {
       [ (gogoproto.casttype) =
             "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
 }
+
+message CommandBatchSignCompleted {
```

Comment:
Not a strong opinion, but I'd prefer the naming of these events to be consistent with the status of the object, e.g. command batch. So that would make `CommandBatchSigned` and `CommandBatchAborted`.